AutoMad/storySaver.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Oct 21 15:44:54 2024
+
+@author: dragon
+"""
+
+import os.path
+import random
+
+import numpy as np
+import pandas as pd
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly","https://www.googleapis.com/auth/documents"]
+SAVEDOC = "1Oha700Q9R_gGvVT2G2NYdL8Ixpr0xGwNVDJ3jMWOOhI"
+
+def saveStory(story):
+    story=story.replace("<br>","\n")+"\n\n\n"
```

Comment:
Is this still appropriate with moving the `<br>` replacement into the web frontend?

Reply:
Yes in this case, I'm pulling from the front end so that I don't have to save story state. There is a chance someone could generate a story and then try to save it after a server reboot.